### PR TITLE
README: update the custom repo URL to github

### PR DESCRIPTION
**Pull request by avlitman**
Since we moved to github we are not using jenkins.

Signed-off-by: Aviv Litman <alitman@redhat.com>

### CODE REVIEW

**Review by tinez — APPROVED** [review]

**Review by tinez — CHANGES_REQUESTED** [review]
Please also mention the necessity to have the `GITHUB_TOKEN` env variable defined

**Comment by avlitman** [comment]
> Please also mention the necessity to have the `GITHUB_TOKEN` env variable defined

Done.

File: README.md [hunk 1]
```diff
@@ -44,9 +44,12 @@ If you have any other questions, please join [oVirt Development forum / mailing
 Make sure your machine is set up with `setup_for_ost.sh`
 You can use `ost.sh` for running the complete suite on a concrete ost-images distro
 E.g. `./ost.sh run basic-suite-master el8stream`
-You can use `--custom-repo` for verifying an oVirt patch
-E.g `./ost.sh run basic-suite-master el8stream --custom-repo=https://jenkins.ovirt.org/job/ovirt-dwh_standard-check-patch/****/`
-You can pass this option multiple times to test more than 1 build at once.
+
+You can use `--custom-repo` for verifying an oVirt patch:
+
+*On your GitHub account go to Settings/Developer settings/Personal access tokens, and generate the token with appropriate permissions.
```
Comment (tinez) [inline README.md:50]: The formatting here doesn't work as you'd expect I think - please have a look at the rich diff

Comment (tinez) [reply README.md:50]: re: `appropriate permissions` personally I always forget which permissions are needed to make it work - I'd really appreciate if they were mentioned explicitly.

**Review by tinez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (avlitman) [reply README.md:50]: > 

I used the format from lines 72-75, I want it to be a list, I can see a preview somehow ?

Comment (avlitman) [reply README.md:50]: > re: `appropriate permissions` personally I always forget which permissions are needed to make it work - I'd really appreciate if they were mentioned explicitly.

In my account under 'Select scopes' I selected the 'repo' (include all) and the 'workflow':
Screenshot from 2022-03-01 14-55-08

but really not sure if less/more is needed, can you check your permissions?

Comment (avlitman) [reply README.md:50]: @tinez So it works with the 'repo'  permissions, you prefer to verify as well ? or I can add that only 'repo' are needed?

**Review by tinez — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tinez) [reply README.md]: I did verify that too - `repo` seems to suffice

**Review by tinez — APPROVED** [review]

**Pull request merged**